src/components/mapPageComponents/mobileMapPage/index.tsx

```diff
@@ -14,18 +14,24 @@ const MainContent = styled.div`
 type MobileMapPageProps = {
   readonly blocks: BlockGeoData;
   readonly neighborhoods: NeighborhoodGeoData;
+  readonly sites: SiteGeoData;
 };
 
 const MobileMapPage: React.FC<MobileMapPageProps> = ({
   blocks,
   neighborhoods,
+  sites,
   children,
 }) => (
   <>
     <MainContent>
       <PageLayout>
         <Content>
-          <MapView blocks={blocks} neighborhoods={neighborhoods} />
+          <MapView
```

Comment:
❓ Will the blocks and the sites both be passed to the map view on the reservation and tree maps?

Reply:
Yeah, I was thinking the map view would have all the data and we'd toggle visibility based on another prop indicating map view. This is a good point though, for adding the tree map I'll see if it's easier to just not pass one or the other.